### DISCUSSION TITLE
Organização de Branches

### DISCUSSION THREAD

**Discussion by diogofaria73** [diego3g/rsxp-2023#42] [General]
Não encontrei nada explicando como será a organização de branches do projeto, alguem saberia me dizer?

**Comment by rpc1910** [comment]
Não sei se já foi definido, mas acho que poderíamos utilizar o Git Flow -> https://danielkummer.github.io/git-flow-cheatsheet/index.pt_BR.html

**Comment by leoferreiralima** [comment]
Entendo que com um CI setado no ambiente as branchs aqui poderiam ser restrigindas a main mesmo.

No nosso fork criamos a branch como quisermos e abrimos a PR sempre para a main.

Garantindo que todo o código esta coberto por tests acredito que não teriamos problemas.

Como @diego3g disse em alguma discussão acredito que deixar o mais simples possível por enquanto e caso haja necessidade vamos evoluindo

**Reply by diego3g** [reply]
Isso! Não quero complicar, vamos trabalhar apenas com a `main` mesmo e no máximo pensemos em um branch de `staging` no futuro.

**Comment by diego3g** [comment]
Não quero complicar, vamos trabalhar apenas com a main mesmo e no máximo pensemos em um branch de staging no futuro.

Vou fechar essa discussão já que já tem bastante discussão aberta e to quase me perdendo aqui 😂
